README.md

```diff
@@ -2,20 +2,41 @@
 
 [![CircleCI](https://circleci.com/gh/DataDog/pupernetes.svg?style=svg)](https://circleci.com/gh/DataDog/pupernetes) [![Build Status](https://travis-ci.org/DataDog/pupernetes.svg?branch=master)](https://travis-ci.org/DataDog/pupernetes) [![Go Report Card](https://goreportcard.com/badge/github.com/DataDog/pupernetes)](https://goreportcard.com/report/github.com/DataDog/pupernetes)
 
+## Table of Contents
+- [Goals](#goals)
+- [Requirements](#requirements)
+  * [Runtime](#runtime)
+    * [Executables](#executables)
+    * [Systemd](#systemd)
+    * [Resources](#resources)
+  * [Development](#development)
+    * [Build](#build)
+- [Getting started](#getting-started)
+  * [Download](#download)
+  * [Run](#run)
+  * [Stop it](#stop-it)
+  * [Systemd as job type](#systemd-as-job-type)
+  * [Command line docs](#command-line-docs)
+- [Metrics](#metrics)
+- [Current limitations](#current-limitations)
+
+## Goals
+
 Run a managed Kubernetes setup.
 
 This project purpose is to provide a simple Kubernetes setup to validate any software on top of it.
 
-You can use it to validate a software dependence on Kubernetes itself or just to run some classic app workflows with [argo](https://github.com/argoproj/argo)
+You can use it to validate a software dependence on Kubernetes itself or just to run some classic app workflows with [argo](https://github.com/argoproj/argo).
+
+As pupernetes runs in [travis](./.travis.yml) and [circle-ci](./.circleci/config.yml), it becomes very easy to integrate this tool in any Kubernetes project.
 
-Our main use case is the end to end testing pipeline of the [datadog-agent](https://github.com/DataDog/datadog-agent)
+Our main use case is the end to end testing pipeline of the [datadog-agent](https://github.com/DataDog/datadog-agent).
```

Comment:
Suggestion: This project has been initially designed to perform the end to end testing of the [datadog-agent](https://github.com/DataDog/datadog-agent).